scanners/boostsecurityio/baseline/module.yaml

```diff
@@ -0,0 +1,12 @@
+api_version: 1.0
+
+
+id: boostsecurityio/scanner
+name: BoostSecurity Scanner
+namespace: boostsecurityio/scanner
+group: baseline
```

Comment:
So far, the "path" to the group, the technically should be the id, will be `boostsecurity/groups/baseline`